package.json

```diff
@@ -2,11 +2,12 @@
   "name": "csde-components-boilerplate",
   "version": "1.8.0",
   "description": "Boilerplate project for creating Studio Digital Editor components.",
-  "main": "index.js",
+  "type": "module",
+  "main": "index.mjs",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "gulp",
-    "dev": "gulp dev",
+    "build": "node index.mjs",
```

Comment:
Was thinking if we need another command to build a specific component set, but probably not important for now.

Reply:
I think we indeed need it, but was thinking we should add it in the next step.